src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -40,9 +40,21 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->children()
-                ->scalarNode('title')->defaultValue('')->info('The title of the API.')->end()
-                ->scalarNode('description')->defaultValue('')->info('The description of the API.')->end()
-                ->scalarNode('version')->defaultValue('0.0.0')->info('The version of the API.')->end()
+                ->scalarNode('title')
+                    ->info('The title of the API.')
+                    ->cannotBeEmpty()
+                    ->defaultValue('')
```

Comment:
isn't it useless given the node that cannot be empty?

Reply:
In fact we must allow empty strings, but not `null`. Maybe should be just normalize empty to `''`?

Reply:
IMHO, the config here is simpler than normalizing everything ...

Reply:
but forbids empty strings

Reply:
I agree we must allow empty strings or it will be a big BC break (for instance the Flex recipe doesn’t set those keys).